netlify/functions/0018_template_to_esign_document.js

```diff
@@ -29,30 +41,47 @@ export const handler = async (event) => {
       };
     }
 
-    const uploadRes = await fetch(
-      "https://api.firma.dev/functions/v1/signing-request-api/signing-requests",
-      {
-        method: "POST",
-        headers: {
-          Authorization: API_KEY,
-          "Content-Type": "application/json",
-        },
-        body: JSON.stringify({
-          name: fileName,
-          description: fileName,
-          document: fileData.replace(/^data:.*;base64,/, ""),
-          settings: {
-            allow_editing_before_sending: true,
-            attach_pdf_on_finish: true,
-            allow_download: true,
-          },
-        }),
-      },
+    const generatedFields = populateApiFields(fields);
+    const generatedPropertyOwnerFields = populateSignerFields(
+      "Property_Owner",
+      generatedFields.owner,
+      generatedFields.ownerEmail,
+    );
+    const generatedTenantFields = populateSignerFields(
+      "Tenant",
+      generatedFields.tenant,
+      generatedFields.tenantEmail,
     );
 
+    const uploadRes = await fetch(GoodSignTemplateToEsignUrl, {
+      method: "POST",
+      headers: {
+        Authorization: API_KEY,
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        uuid: uuid,
+        doc_name: doc_name,
+        attachment_names_in_order: [],
```

Comment:
i'm not really sure what we are going to do for the attachments, however, we should kind of add some metadata to our files, so that these metadata can also be accessed after the file has been read.